skellytracker/trackers/openpose_tracker/openpose_tracker.py

```diff
@@ -132,27 +132,13 @@ def process_video(
 if __name__ == "__main__":
     # Example usage
 
-    input_video_folder = Path(
-        r"C:\Users\aaron\FreeMocap_Data\recording_sessions\freemocap_sample_data"
-    )
-    input_video_filepath = (
-        input_video_folder
-        / "synchronized_videos"
-        / "sesh_2022-09-19_16_16_50_in_class_jsm_synced_Cam1.mp4"
-    )
-
-    output_video_filepath = input_video_folder / "openpose_annotated_videos"
-    output_video_filepath.mkdir(parents=True, exist_ok=True)
 
-    output_json_path = (
-        input_video_folder / "output_data" / "raw_data" / "openpose_jsons"
-    )
+    openpose_root_folder_path = r"C:\openpose"
+    output_json_path = Path(r'C:\openpose\output_json')
```

Comment:
This path is wrong - the we want the output jsons to be saved within the `raw_data` folder of the FreeMoCap recording it's running on - not the actual OpenPose demo folder. 

This does bring up a point I was thinking about though. Functionally, I realized that this particular parameter shouldn't be a parameter that users change (the `openpose_jsons` folder should always be in the `raw_data` folder). 

The reason this is a little tricky is because, as far as I understand, none of the other trackers we have so far need to do file saving or management *during* their run time processes - we pass the data back out to the `process_folder_full_of_videos` function to take care of that, but for OpenPose we do need to do file I/O, both to save and load the json files. So we need want a way to do some file management specific to the OpenPose tracker. Should we take a version of this code (that's in `process_folder_full_of_videos`):

```
    if output_folder_path is None:
        output_folder_path = (
            synchronized_video_path.parent / "output_data" / "raw_data" / file_name
        )
    else:
    ```
    
    and put it in the `openpose_tracker`, but making that `openpose_jsons` folder instead?

Reply:
I took a stab at implementing this. Let me know if it works and particularly if the path specified is correct (`Path(input_video_filepath).parent.parent / "output_data" / "raw_data" / "openpose_jsons"`)